### PR TITLE
CO-1574 Add auto refresh for communication revisions

### PR DESCRIPTION
**Pull request by popod**

### CODE REVIEW

File: partner_communication_revision/README.rst [hunk 1]
```diff
@@ -6,7 +6,14 @@ Partner Communication Revisions
...
 
-Usage
+Installation
+============
+
+You need to add regex library
+
+- sudo pip install regex
+
+Usagers
```
Comment (ecino) [inline partner_communication_revision/README.rst]: Usage not Usagers

File: partner_communication_revision/models/communication_keyword.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 import re
 
 from odoo import api, models, fields
-
+# from . import communication_revision as test
```
Comment (ecino) [inline partner_communication_revision/models/communication_keyword.py]: Remove this

File: partner_communication_revision/models/communication_keyword.py [hunk 3]
```diff
@@ -236,6 +236,9 @@ def toggle_edit_value(self):
         for keyword in set_false:
             vals['edit_changed'] = keyword.edit_changed + 1
             keyword.write(vals)
+
+        # self.env['partner.communication.revision'].browse(self.revision_id.id).refresh_text()
```
Comment (ecino) [inline partner_communication_revision/models/communication_keyword.py]: Remove this

**Review by ecino — CHANGES_REQUESTED** [review]

**Comment by popod** [comment]
Has worked, but does not works anymore :(

**Comment by ecino** [comment]
Maybe you can comment again the import of the xml loading the javascript in the manifest file, so that the js is not executed. But we can keep your code in the case we could use it in the future.

**Review by ecino — APPROVED** [review]

**Pull request merged**